.changeset/eight-needles-lay.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I don't think `web` needs a changeset, or at least we haven't been using them there yet.

Also, to keep the changelog a bit more clean, can the starters have their own changeset?